### ISSUE TITLE
Include all multilingual images in json

### ISSUE THREAD

**Issue by aleene** [openfoodfacts/openfoodfacts-server#460]
The world read json does not include all urls for the images, it only contains the images for a single language. This makes it very hard to create apps that support multiple languages.

Include the urls of the image urls for the other languages as well.

**Comment by teolemon** [comment]
For reference
https://github.com/aleene/FoodViewer/issues/168

**Comment by hangy** [comment]
What do you think of this structure? Or would it be better to have front/nutrition/ingredients at a higher level and the languages at the lower level? I am not really decided on that. :) ❔ 
```
{
   "code":"4311501452998",
   "status":1,
   "status_verbose":"product found",
   "product":{
      "selected_images":{
         "de":{
            "front":{
               "display":"http://static....",
               "small":"http://static....",
               "thumb":"http://static...."
            }
         },
         "fr":{
            "nutrition":{
               "display":"http://static....",
               "small":"http://static....",
               "thumb":"http://static...."
            },
            "ingredients":{
               "display":"http://static....",
               "small":"http://static....",
               "thumb":"http://static...."
            },
            "front":{
               "display":"http://static....",
               "small":"http://static....",
               "thumb":"http://static...."
            }
         }
      }
   }
}
```

**Comment by teolemon** [comment]
cc @stephanegigandet

**Comment by aleene** [comment]
I think I would put the different languages lower, i.e. under ingredients, nutrition, front. Seems more logic. Even under display/small/thumb.
I intend to store the urls for a specific image as a dictionary with the langauges as keys. So for me, it would be:
```
{
   "code":"4311501452998",
   "status":1,
   "status_verbose":"product found",
   "product":{
      "selected_images":{
         "front":{
               "display":{
                     "de":"http://static....",
                     "fr":"http://static....",
                },
               "small":{
                 "de":"http://static...."
               },
               "thumb":{
                  "fr":"http://static...."
               },
         },
         "nutrition":{
               "display":{
                   "fr":"http://static...."
                },
               "small":{
                   "fr":"http://static...."
               },
               "thumb":{
                    "fr":"http://static...."
                },
            },
            "ingredients":{
               "display":{
                     "fr":"http://static...."
                },
               "small":{
                        "fr":"http://static...."
                },
               "thumb":{
                      "fr":"http://static...."
                }
            },
            "front":{
               "display":{
                   "fr":"http://static...."
                },
               "small":{
                    "fr":"http://static...."
               },
               "thumb":{
                    "fr":"http://static...."
                }
            }
         }
      }
   }
}
```

**Comment by hangy** [comment]
I've implemented this in 68683c3cb2a5c7cd9e4ffa6940fd64a9d30478a5 and opened [a PR](https://bitbucket.org/openfoodfacts/product-opener/pull-requests/54) for review.
